README.md

```diff
@@ -1,254 +1,422 @@
-```
+    To leverage the Conjur Buildpack so that secret values are automatically
+    injected into your application's environment at runtime, your application needs
+    a `secrets.yml` file is. The `secrets.yml` file gives a mapping of **environment variable name**
+    to a **location where a secret is stored in Conjur**. For more information about
+    creating this file, [see the Summon documentation](https://cyberark.github.io/summon/#secrets.yml).
 
-In PCF version 2.0+, when the service broker creates the Host identity for your application
-in Conjur, it will automatically add it to a Conjur Layer representing the `Organization`
-and `Space` the application is deployed into on PCF. This allows `Spaces` to be privileged
-and any apps deployed into that space to inherit those privileges.
+2. **Bind Your Application to the Conjur Service**
 
+    Binding your application to the `conjur` service instance provides the application with an
+    identity in Conjur, and credentials that it may use to retrieve secrets.
 
-### Update Conjur Policy to Privilege Your Application
+    - **Using the Cloud Foundry CLI**
 
-Once your app has a Host identity in Conjur, you can update Conjur policy to add
-entitlements for the app to access secret values in Conjur. The host identity of
-the application is stored in the `authn_login` field in the `cyberark-conjur`
-credentials in the application's environment, and might look something like
-`host/cf/prod/0299a19d-7de4-4e98-89f6-372ac7c0521f` (for example, if your `CONJUR_POLICY`
-was set to `cf/prod`).
+        To bind your application to the `conjur` service using the CLI, run the command:
+        ```
+        cf bind-service my-app conjur
+        ```
 
-In PCF version 2.0+, the host identity will include the Organization and Space GUIDs in
-the Host identity, for example:
-```
-host/cf/prod/cbd7a05a-b304-42a9-8f66-6827ae6f78a1/8bf39f4a-ebde-437b-9c38-3d234b80631a/c363669e-e43b-40b9-b650-493d3bdb4663
-```
+    - **Using the Application Manifest**
+
+        Alternatively you can specify the conjur service in your application manifest:
+        ```
+        ---
+        applications:
+        - name: my-app
+          services:
+          - conjur
+        ```
+
+    #### Application vs Space Host Identity
+
+    When an application is bound to the Conjur service, it receives an identity in Conjur
+    and credentials to authenticate to Conjur.
+
+    The service broker may be configured to either create a single Conjur identity shared by
+    all applications in a space, or to create a Conjur identity for each application
+    separately.
+
+    In PCF version 2.0+, when the service broker creates the identity for your application
+    in Conjur, it automatically adds it to a Conjur Layer representing the `Organization`
+    and `Space` where the application is deployed. These layers may be used for control secret
+    access at the org or space level, rather than the application host itself.
```

Comment:
Is it worth having an example here to explain what we mean by this? I want to make sure people understand the difference between entitling at the app-host / space-host level, vs the space layer, vs the org layer. Is there an entitlements or layer page we can reference in the Conjur docs?

Reply:
:) you have an example below. I wonder if that could be referred to here in that case ("for more details see below")